internal/service/contentprovider/moduleconfig_test.go

```diff
@@ -44,14 +44,14 @@ func Test_ModuleConfig_GetDefaultContent_ReturnsError_WhenRequiredArgsMissing(t
 		{
 			argName: contentprovider.ArgModuleVersion,
 			args: types.KeyValueArgs{
-				contentprovider.ArgModuleName:    "module-name",
```

Comment:
This arg still will be the `module-name`. Why rename it to` create-name`?

Reply:
This was a refactoring error, reverted.